.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  env: {
+    node: true,
+    commonjs: true,
+    es2021: true,
+    jest: true,
+  },
+  extends: 'eslint:recommended',
+  parserOptions: {
+    ecmaVersion: 13,
+  },
+  plugins: ['anti-trojan-source', 'jest'],
+  rules: {
+    /**
+     * Halt if a trojan source attack is found
+     * https://github.com/lirantal/eslint-plugin-anti-trojan-source
+     */
+    'anti-trojan-source/no-bidi': 'error',
+  },
+  overrides: [
+    {
+      files: ['**/test.js'],
+      rules: {
+        'no-unused-vars': 'off',
```

Comment:
Why do we need this?

Reply:
In the `test.js` file, there are no-unused-vars errors for variables at places like:
```js
((req, user) => { ... })
```
Even though the variables in such instances are technically unused, I felt that they do hold meaning. I looked into the options for the [no-unused-vars](https://eslint.org/docs/rules/no-unused-vars) rule but there's no suitable option that would avoid such cases in `test.js`. So, rather than adding `//eslint-disable-next-line` for each such error, I added that override for the `test.js` file

Reply:
Let's disable this globally in the test.js file via a comment at the top vs. doing it for the whole project.

Reply:
Done!